index.rst

```diff
@@ -10,20 +10,849 @@
+
+
+- ~4 cores per CCD are required to perform the data processing
+- Using the full 189 CCDs also requires 756 cores which is nearly the entire Antu capability (784 cores)
+- To support required data Input/Output (I/O), a cluster would ideally have a small number of cores per node, then spread the data out across multiple disks.
+  Antu has a high core-to-node ratio, and is therefore likely unable to run Rapid Analysis for the entire array at a ~30s cadence.
+
+
+At the moment, it is unclear if the computing infrastructure could be augmented to support full-frame on-the-fly processing in Chile.
+If not, then the remaining option is to reduce the number of CCDs that get processed.
+DECam encountered the same constraints and invoked a pipeline that supports different configurations that specify various patterns of sensors to reduce.
+For example, pointing tests used just the central portion of the focal plane.
+A list of possible focal plane configurations should be created; we have already reached out to the AOS[#]_ and Science Verification[#]_ groups for suggestions.
+It is recommended that Rubin adopt a similar architecture as it is not expected that any summit-based Rapid Analysis image quality metrics would require the full array.
+Especially since the camera diagnostic cluster handles the low-level health checks for all sensors, as is discussed in `Deliverable 5`_.
+
+The University of Washington group is now investigating the SFP performance enhancements.
+Scaling the experience gained with LATISS, it is expected that a 30s image cadence is feasible and the primary speed limitation will be the I/O constraints.
+
+.. [#] The AOS group has already communicated that a checkerboard pattern for the focal plane, while omitting the 8 outermost sensors which are highly vignetted, is satisfactory to accomplish their analysis requirements.
+
+.. [#] The Science Verification group has indicated that full-frame on-the-fly processing is not required, so long as full frame processing occurs at the USDF within 24-hours.
+
+.. _analysis_tools_overview:
+
+analysis_tools
+^^^^^^^^^^^^^^
+
+Several `basic per-detector data quality statistics <https://confluence.lsstcorp.org/display/LSSTCOM/Science+performance+metrics+to+support+nightly+operations>`_ are generated during SFP and persisted in the Butler repository.
+These basic quantities can be supplemented by more detailed data quality diagnostics produced by other Science Pipeline components.
+
+The recently released `analysis_tools python package <https://github.com/lsst/analysis_tools>`_ is a refactor of the faro and analysis_drp packages that provides both metric and plot generation functionality.
+The package includes a set of analysis modules that can be run as Tasks within a data reduction pipeline, as part of a separate afterburner pipeline, or imported and executed within a standalone in a script/notebook.
```

Comment:
or imported and executed within a standalone in a script/notebook -> or imported and executed standalone, in a script/notebook